sklearnex/tests/test_patching.py

```diff
@@ -122,8 +122,17 @@ def test_standard_estimator_patching(caplog, dataframe, queue, dtype, estimator,
     with caplog.at_level(logging.WARNING, logger="sklearnex"):
         est = PATCHED_MODELS[estimator]()
 
-        if dtype == np.float16 and queue and not queue.sycl_device.has_aspect_fp16:
-            pytest.skip("Hardware does not support fp16 SYCL testing")
+        # Add skips for various issues in sklearnex, additions here require tickets.
+        if queue:
+            if dtype == np.float16 and not queue.sycl_device.has_aspect_fp16:
+                pytest.skip("Hardware does not support fp16 SYCL testing")
+            elif queue.sycl_device.is_gpu and estimator in [
+                "KMeans",
+                "ElasticNet",
+                "Lasso",
+                "Ridge",
+            ]:
+                pytest.skip(f"{estimator} does not support GPU queues")
```

Comment:
Why we are removing them from GPU testing?

Reply:
https://github.com/intel/scikit-learn-intelex/pull/1737#issuecomment-2046586614 written here